src/js/index-android.js

```diff
@@ -9,13 +9,19 @@
 
 const inAppPurchase = { utils };
 
+const createIapError = (reject) => {
+  return (err = {}) => {
+    err.errorCode = err.code;
+    delete err.code;
```

Comment:
I actually delete the `code` property here because it looked confusing to have both. My friend made a really good point though that this would force you to bump the major version to indicate a breaking change. I'll let you decide, but maybe both properties should exist until you are ready for a major revision?